src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -386,11 +386,31 @@ private void OnHostExited(HostProviderEventArgs e)
             }
         }
 
+        private void WarnIfDebugEnvironment()
+        {
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("VSTEST_HOST_NATIVE_DEBUG")))
+            {
+                EqtTrace.Warning("VSTEST_HOST_NATIVE_DEBUG is set and will cause testhost to hang.");
```

Comment:
I don't think "hang" is the correct word to use here. Hang suggests that this is a bad thing (at least to me). And it does not suggest how to "solve" it. I would rather see it say: and will cause testhost to wait indefinitely for a debugger to be attached.

This message should also be localized.

The same treatment should be given to datacollector, because it can do the same thing.